x-pack/plugins/cases/server/services/user_actions/test_utils.ts

```diff
@@ -52,7 +52,7 @@ export const createUserActionFindSO = (
 });
 
 export const createConnectorUserAction = (
-  overrides?: Partial<CaseUserActionWithoutReferenceIds>
+  overrides?: Partial<SavedObject<CaseUserActionWithoutReferenceIds>>
```

Comment:
@elastic/response-ops-cases, the overrides from these utils are unused, but adding a `created_by` on a root saved object found a type issue with the types